util/dvsim.py

```diff
@@ -81,7 +81,14 @@ def resolve_branch(arg_branch):
 
 # Get the project root directory path - this is used to construct the full paths
 def get_proj_root():
-    return os.path.abspath(os.path.dirname(os.path.realpath(__file__)) + "/..")
+    result = subprocess.run(["git", "rev-parse", "--show-toplevel"],
+                             stdout=subprocess.PIPE)
+    proj_root = result.stdout.decode("utf-8").strip()
+    if proj_root == "":
+        log.error(
+            "Failed to find proj_root.")
```

Comment:
Perhaps provide more context: `("Attempted to find the root directory of this GitHub repository by running:\n".." \nBut the command appears to have failed: \n%s", result.stderr.decode("utf-8")`

Reply:
Done, thanks for the suggestion